package.json

```diff
@@ -174,6 +174,8 @@
 		"test:php": "node ./tools/local-env/scripts/docker.js run -T php composer update -W && node ./tools/local-env/scripts/docker.js run php ./vendor/bin/phpunit",
 		"test:e2e": "node ./tests/e2e/run-tests.js",
 		"test:visual": "node ./tests/visual-regression/run-tests.js",
-		"wp-packages-update": "wp-scripts packages-update"
+		"wp-packages-update": "wp-scripts packages-update",
```

Comment:
I guess we can remove `wp-packages-update` in favor of new scripts. It played the same purpose.

Reply:
Not quite, the process takes three steps:

1. `wp-packages-update`: Install the latest version of the packages already listed in package.json
2. `wp-packages-refresh-deps`: Install any new `@wordpress` packages that are now required
3. `wp-packages-refresh-deps`: Update any non-`@wordpress` deps to the same version as required in the `@wordpress` packages (e.g. react 16 -> 17)

We could bring 1 into `wp-packages-refresh-deps`, but that means maintaining that task in two places. I'd rather reuse the code we already have in Gutenberg, no matter how similar it is to the other two.

Reply:
You can call `wp-scripts packages-update` directly, right?

Reply:
ah yes, that's correct